apps/test/unit/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/resultsTest.js

```diff
@@ -1,67 +1,89 @@
 import {expect} from 'chai';
 import Results from '@cdo/apps/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results';
-import {shallow} from 'enzyme';
+import {mount} from 'enzyme';
 import React from 'react';
 
 describe("Local Summer Workshop Daily Survey Results class", () => {
   it("Renders expected number of sessions, questions, and free responses", () => {
-    const results = shallow(
+    const results = mount(
       <Results
         questions={{
             'day_1': {
-              'q1': {text: 'Day 1, Question 1'},
-              'q2': {text: 'Day 1, Question 2'},
-              'q3': {text: 'Day 1, Question 3'},
-              'f1': {text: 'Day 1, Free Response 1', free_response: true},
-              'f2': {text: 'Day 1, Free Response 2', free_response: true}
+              'general': {
+                'q1': {text: 'Day 1, Question 1'},
+                'q2': {text: 'Day 1, Question 2'},
+                'q3': {text: 'Day 1, Question 3'},
+                'f1': {text: 'Day 1, Free Response 1', free_response: true},
+                'f2': {text: 'Day 1, Free Response 2', free_response: true}
+              }
             },
             'day_2': {
-              'q4': {text: 'Day 2, Question 1'},
-              'q5': {text: 'Day 2, Question 2'},
-              'f3': {text: 'Day 2, Free Response 1', free_response: true}
+              'general': {
+                'q4': {text: 'Day 2, Question 1'},
+                'q5': {text: 'Day 2, Question 2'},
+                'f3': {text: 'Day 2, Free Response 1', free_response: true}
+              },
+              'facilitator': {
+                'f_q1': {text: 'Facilitator specific question 1'},
+                'f_f1': {text: 'Facilitator specific free response question 1', free_response: true}
+              }
             }
         }}
         thisWorkshop={{
           'day_1': {
-            'q1': 3.5,
-            'q2': 3.6,
-            'q3': 3.7,
-            'f1': ['a', 'b', 'c'],
-            'f2': ['d', 'e', 'f']
-          },
-          'day_2': {
-            'q4': 3.8,
-            'q5': 3.9,
-            'f3': ['g', 'h', 'i']
-          }
-        }}
-        allMyWorkshops={{
-          'day_1': {
-            'q1': 2.5,
-            'q2': 2.6,
-            'q3': 2.7,
-            'f1': ['Should not render']
+            'general': {
+              'q1': 3.5,
+              'q2': 3.6,
+              'q3': 3.7,
+              'f1': ['a', 'b', 'c'],
+              'f2': ['d', 'e', 'f']
+            }
           },
           'day_2': {
-            'q4': 2.8,
-            'q5': 2.9
+            'general': {
+              'q4': 3.8,
+              'q5': 3.9,
+              'f3': ['g', 'h', 'i']
+            },
+            'facilitator': {
+              'f_q1': {
+                '1': 1.5,
+                '2': 1.6
+              },
+              'f_f1': {
+                '1': ['j', 'k'],
+                '2': ['l', 'm']
+              }
```

Comment:
It would be simpler to organize this by survey, so for each day we'd have 1 set of general questions, and a set of facilitator-specific questions per facilitator. This appears to be collated by question, with each facilitator's response spread out, but I can't really tell with the labels.